ui/packages/consul-ui/app/models/topology.js

```diff
@@ -11,7 +12,30 @@ export default class Topology extends Model {
   @attr('string') Namespace;
   @attr('string') Protocol;
   @attr('boolean') FilteredByACLs;
+  @attr('boolean') TransparentProxy;
+  @attr('boolean') DefaultAllow;
+  @attr('boolean') WildcardIntention;
   @attr() Upstreams; // Service[]
   @attr() Downstreams; // Service[],
   @attr() meta; // {}
+
+  @computed('Upstreams', 'Downstreams')
+  get undefinedIntention() {
```

Comment:
I'm using `undefinedIntention` not just `undefined`. `hasIntention` is the opposite of what I'm trying to figure out in this compute.

Reply:
In that case I think this would be preferable: `!item.hasIntention`/`(not item.hasIntention)`.

I'd say 'generally' boolean things like this should be positively named (mostly with `is` or `has`), and then use either `!` or `(not)` to flip it if/when you need to. Otherwise at some point you end up writing double negatives like `!item.undefinedIntention`/`(not item.undefinedIntention)` which is like saying "doesn't doesn't have an intention" or "doesn't not have an intention", and things can start to get confusing.